Core/IResizable.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace StarlightRiver.Core
+{
+	interface IResizable
+	{
+		public void ResizeTarget() { }
+
+		public bool IsResizable { get; }
```

Comment:
Why would something ever be an IResizable but not be resizable? Is this more intended as a 'should resize' switch? If it is this name seems a bit in conflict with the name of the interface